File: VB/10_Conversion/SaveSvgWithOption.vb
Imports System.IO
Imports Spire.Presentation
Imports Spire.Presentation.Drawing

Namespace SaveSvgWithOption
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			Dim inputFile As String = "..\..\..\..\..\..\Data\SaveSvgWithOption.pptx"

			' Create Presentation object and load the file
			Dim ppt As New Presentation()
			ppt.LoadFromFile(inputFile)

			' Save the underline as decoration when converting to Svg
			ppt.SaveToSvgOption.SaveUnderlineAsDecoration = True

			' Save to Svg
			Dim svgByte() As Byte = ppt.Slides(0).Shapes(0).SaveAsSvgInSlide()
			Dim fs As New FileStream("SaveSvgWithOption" & "1.svg", FileMode.Create)
			fs.Write(svgByte, 0, svgByte.Length)
			fs.Close()

			'Dispose
			ppt.Dispose()

		End Sub

	End Class
End Namespace

File: CS/10_Conversion/SaveSvgWithOption.cs
using System;
using System.Drawing;
using System.IO;
using System.Windows.Forms;
using Spire.Presentation;
using Spire.Presentation.Drawing;

namespace SaveSvgWithOption
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {            
            string inputFile = @"..\..\..\..\..\..\Data\SaveSvgWithOption.pptx";

            // Create Presentation object and load the file
            Presentation ppt = new Presentation();
            ppt.LoadFromFile(inputFile);

            // Save the underline as decoration when converting to Svg
            ppt.SaveToSvgOption.SaveUnderlineAsDecoration = true;

            // Save to Svg
            byte[] svgByte = ppt.Slides[0].Shapes[0].SaveAsSvgInSlide();
            FileStream fs = new FileStream("SaveSvgWithOption" + "1.svg", FileMode.Create);
            fs.Write(svgByte, 0, svgByte.Length);
            fs.Close();

            //Dispose
            ppt.Dispose();

        }

    }
}